### PR TITLE
Extend changelog of ActiveModel#errors for Ruby 3

### LINKED ISSUE #41270: wrong number of arguments (given 3, expected 1..2) on errors.add when given hash
### Steps to reproduce

Use errors.add in your model to add validation errors:

```rb
class User < ApplicationRecord
  def add_error
    errors.add(:name, :invalid, {})
  end
end

User.new.add_error
```

### Expected behavior

Not throwing an error (which is Rails 6.0 behaviour)

### Actual behavior

`wrong number of arguments (given 3, expected 1..2)` on Rails 6.1

This boils down to a changed method signature between 6 and 6.1

6.0: https://api.rubyonrails.org/v6.0.0/classes/ActiveModel/Errors.html#method-i-add
6.1: https://api.rubyonrails.org/v6.1.1/classes/ActiveModel/Errors.html#method-i-add

The change was introduced in a three years old commit: https://github.com/rails/rails/commit/d9011e39357300fe78720227af4c13b4bc4ac4dd (quite some archeology here 🦖 )

The fix for consumer side is rather easy:

```rb
errros.add(:name, invalid, **{})
```

Not sure what the Rails team can do here, but that breaking change definitely caught me by surprise on one project, since it's not mentioned at all in the [6.1 changelog for ActiveModel](https://github.com/rails/rails/blob/6-1-stable/activemodel/CHANGELOG.md). If nothing else, this issue post here should allow people to find if they run into it (which they will with current versions of kt-paperclip, since this is how I found the issue)

Thanks for reading and keep on the good work!

### System configuration
**Rails version**: 6.1.1

**Ruby version**: 3.0.0

### PR DESCRIPTION
**Pull request by intrip**
### Summary

Add information about Ruby 3 incompatibilities related to kwargs changes.

Closes #41270

### CODE REVIEW

File: activemodel/CHANGELOG.md [hunk 1]
```diff
@@ -79,6 +79,9 @@
     some edge cases won’t be covered, like `errors#first` will return `ActiveModel::Error` and manipulating
     `errors.messages` and `errors.details` hashes directly will have no effect. Moving forward,
     please convert those direct manipulations to use provided API methods instead.
+    Please note that `errors#add` now accepts `options` as keyword arguments instead of `Hash` which
+    introduces a breaking change in Ruby 3 due to [changes to keyword
```
Comment (eileencodes) [inline activemodel/CHANGELOG.md:83]:
```suggestion
    introduced a change in Ruby 3 to [keyword arguments](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/).
```

**Comment by intrip** [comment]
@eileencodes Thanks! Suggestion applied and rebased.

Eventually I'll become better at writing docs 😅

**Review by guilleiguaran — APPROVED** [review]

File: activemodel/CHANGELOG.md [hunk 2]
```diff
@@ -79,6 +79,9 @@
     some edge cases won’t be covered, like `errors#first` will return `ActiveModel::Error` and manipulating
     `errors.messages` and `errors.details` hashes directly will have no effect. Moving forward,
     please convert those direct manipulations to use provided API methods instead.
+    Please note that `errors#add` now accepts `options` as keyword arguments instead of `Hash` which
+    introduced a change in Ruby 3 to [keyword arguments](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/).
+    arguments](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/).
```
Comment (zzak) [inline activemodel/CHANGELOG.md:84]: Looks like a paste bug was merged 😅 should be fixed in #42644

Comment (intrip) [reply activemodel/CHANGELOG.md:84]: Ouch 🤦 
@zzak thanks for noticing and fixing it!

**Pull request merged**